src/code/z_sram_NES.c

```diff
@@ -237,9 +237,21 @@ u16 D_801C66D0[ARRAY_COUNT(gSaveContext.save.weekEventReg)] = {
     /* 99 */ 0,
 };
 
+// Unused remnant values from OoT. Not the correct sizes in MM.
+#define OOT_SAVECONTEXT_SIZE 0x1428
+#define OOT_SLOT_SIZE (OOT_SAVECONTEXT_SIZE + 0x28)
+#define OOT_SRAM_HEADER_SIZE 0x10
+#define OOT_SLOT_OFFSET(index) (OOT_SRAM_HEADER_SIZE + 0x10 + (index * OOT_SLOT_SIZE))
```

Comment:
2 thoughts here:
1. In conjunction with the other macros curious if these could just be the `NO_FLASH` variants (again keeping the OoT note). Though not quite 100% on this one.
2. Looking at variable/functions names, it seems like it is actually the FLASH variants with the specifier. For example `gSramSlotOffsets` vs `gFlashSaveStartPages`. Should the macros/all variables (would probably include both save buffers) follow the same convention?

Reply:
Could you expand your second thought, I'm not following. `gSramSlotOffsets` is the absolute offsets intended for SRAM saving, which goes unused in MM because it's designed for a FLASHRAM save, which `gFlashSaveStartPages` captures, but it's in terms of pages and not sizes. What convention would be followed to be consistent?

Reply:
Discussed in Discord https://discord.com/channels/688807550715560050/873211000122384415/1091536340262330410